build/spec.c

```diff
@@ -268,6 +268,7 @@ rpmSpec newSpec(void)
     spec->readStack = xcalloc(1, sizeof(*spec->readStack));
     spec->readStack->next = NULL;
     spec->readStack->reading = 1;
+    spec->readStack->lastConditional = LINE_ENDIF;
```

Comment:
This seems like a bit of a hack compared to having that LINE_DEFAULT value, but not a big deal.